src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -1010,13 +1009,21 @@ trait AppAndGroupFormats {
         "dependencies" -> runSpec.dependencies,
         "upgradeStrategy" -> runSpec.upgradeStrategy,
         "labels" -> runSpec.labels,
-        "acceptedResourceRoles" -> runSpec.acceptedResourceRoles,
         "ipAddress" -> runSpec.ipAddress,
         "version" -> runSpec.version,
         "residency" -> runSpec.residency,
         "secrets" -> runSpec.secrets,
         "taskKillGracePeriodSeconds" -> runSpec.taskKillGracePeriod
       )
+
+      if (runSpec.args.nonEmpty) {
```

Comment:
It'll be an empty list and break this assetion 
`(r1 \ "args").asOpt[Seq[String]] should equal (None)` in
https://github.com/mesosphere/marathon/blob/feature/pods/src/test/scala/mesosphere/marathon/api/v2/json/AppDefinitionFormatsTest.scala#L53